make/include/versioning.sh

```diff
@@ -11,7 +11,7 @@ GIT_DESCRIBE=${GIT_DESCRIBE:-$(git describe --always --tags --long)}
 GIT_TAG=${GIT_TAG:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $1 }' )}
 GIT_COMMITS=${GIT_COMMITS:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $2 }' )}
 GIT_SHA=${GIT_SHA:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $3 }' )}
-ARTIFACT_NAME=${ARTIFACT_NAME:-$(basename $(git config --get remote.origin.url) .git | sed s/^hcf-//)}
+ARTIFACT_NAME=${ARTIFACT_NAME:-$(basename $(git config --get remote.origin.url) .git | sed s/^scf-//)}
```

Comment:
The `| sed s/^scf-//` isn't necessary as the reponame is now just `testbrain`.

Reply:
Addressed with commit 86f541fc3d88226050832483b3e5d363fa779f4c